index.html

```diff
@@ -12,7 +12,31 @@
     <link rel="stylesheet" href="assets/css/main-footer.css">
 </head>
 <body>
-  <header class="main-header"></header>
+    <header class="main-header">
+       <div class="main-header-content">
+        <img class="logo" src="/images/logo.png" alt="Digital College">
+            <div class="search-bar">
+                <input placeholder="Pesquisar produto..." class="main-header-search" type="search">
+                <img class="search-lupa" src="/images/Search.svg" alt="">
+            </div>
+            <a href="" class="register-link">Cadastre-se</a>
+            <a href="" class="button" >Entrar</a>
+            <button class="kart-button">
+                <img src="/images/Buy.svg" alt="">
```

Comment:
favor usar nome de imagens em letras minusculas e não usar a / no início.

`/images/Buy.svg` => `images/buy.svg`